docs/user-guide/standard_transformer.md

```diff
@@ -386,6 +387,64 @@ See below for a complete example on how to declare an ordinal encoder
             sedan: '2'
 ```
 
+#### Cyclical Encoder Specification
+Cyclical encoder are useful for encoding columns that has cyclical significance. By encoding such columns cyclically, you can ensure that the values representing the end of a cycle and the start of the next cycle does not jump abruptly. Some examples of such data are:
+
+- Hours of the day
+- Days of the week
+- Months in a year
+- Wind direction
+- Seasons
+- Navigation Directions
+
+The syntax to define an cyclical encoder is as follows:
+```
+cyclicalEncoderConfig:
+  <Encode By>
+```
+
+There are 2 ways to encode the column:
+
+1. By epoch time: Unix Epoch time is the number of seconds that have elapsed since January 1, 1970 (midnight UTC/GMT). By using this option, we assume that the time zone to encode in will be UTC. In order to use this option you only need to define the period of your cycle to encode.
+
+1. By range: This defines the base range of floating point values representing a cycle. For example, one might define wind directions to be in the range of 0 to 360 degrees, although the actual value may be >360 or <0.
+
+To encode by **epoch time**, use the following syntax:
+```
+cyclicalEncoderConfig:
+  byEpochTime:
+    period: HOUR #HOUR, DAY, WEEK, MONTH, QUARTER, HALF, YEAR
+```
+
+To encode by **range**, use the following syntax:
+```
+cyclicalEncoderConfig:
+  byRange:
+    min: FLOAT
+    max: FLOAT
+```
+Do note that the min and max values are Float. The range is inclusive for the min and exclusive for the max, since in a cycle min and max will represent the same phase. For example, you can encode the days of a week in the range of [1, 8), where 8 and 1 both represents the starting point of a cycle. You can then represent Monday 12am as 1 and Sunday 12pm as 7.5 and so on.
+
+See below for complete examples on how to declare a cyclical encoder:
+
+*By epoch time:*
+```
+-encoders: 
+  -name: payday_trend
+   cyclicalEncoderConfig:
+     byEpochTime:
+       period: MONTH
+```
+
+*By range:*
+```
+-encoders: 
+  -name: wind_dir 
+   cyclicalEncoderConfig:
+      byRange:
+        min: 0
+        max: 360
+```
```

Comment:
Thanks for updating the doc!

Anyway, can you also include the input & output example of the encoder, i.e. for epoch time encoder given a list of date & time what is the epoch output.